app/views/admin/versions/_object_desc_and_link.html.haml

```diff
@@ -235,5 +235,5 @@
     - else
       (Organization Deleted)
   - else
```

Comment:
If the else part no longer has code, why do we need to keep it?

Reply:
I am figuring out the else part code @differentreality. Will get back to you soon with the solution. Well I am posting the screenshot for your reference.
![image](https://user-images.githubusercontent.com/24351012/33337101-e5da641c-d497-11e7-975e-8eada042f516.png)

Reply:
In this screenshot the record for ticket (ID 23), no longer shows for which conference this was. 
This used to look like:
Stella Rouzi created new ticket abc in conference tconf0 1 day ago